src/api/index.js

```diff
@@ -467,3 +467,8 @@ export function getEventListener({ name, namespace }) {
   const uri = getTektonAPI('eventlisteners', { name, namespace });
   return get(uri);
 }
+
+export async function fetchDashboardInfo() {
```

Comment:
Function seems pointless, why not just await on getInstallProperties for the caller?

Reply:
Have removed this now